Library/Homebrew/brew.sh

```diff
@@ -444,7 +444,7 @@ fi
 
 if [[ -n "$HOMEBREW_MACOS" || -n "$HOMEBREW_FORCE_HOMEBREW_ON_LINUX" ]]
 then
-  HOMEBREW_BOTTLE_DEFAULT_DOMAIN="https://homebrew.bintray.com"
+  HOMEBREW_BOTTLE_DEFAULT_DOMAIN="https://ghcr.io/v2/homebrew/core"
```

Comment:
What's the v2 part of this btw? Will it ever become v3?

Reply:
It seems to be a Docker/spec version. It seems unlikely to ever change,

Reply:
Ok, makes sense.

My main concern was whether it was stable enough to expose as `root_url` or whether it should be hidden as a implementation detail.